resources/eventing/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: eventing
+description: Kyma Component 'eventing'
+type: application
```

Comment:
That "type" attribute, is it relevant?

Reply:
There are two Helm chart types: (1) application (2) library

Application is the default. I see so far we did not use it.So I will remove it for consistency reasons.

> The type field defines the type of chart. There are two types: application and library. Application is the default type and it is the standard chart which can be operated on fully. The library chart provides utilities or functions for the chart builder. A library chart differs from an application chart because it is not installable and usually doesn't contain any resource objects.
> Note: An application chart can be used as a library chart. This is enabled by setting the type to library. The chart will then be rendered as a library chart where all utilities and functions can be leveraged. All resource objects of the chart will not be rendered.
https://helm.sh/docs/topics/charts/#chart-types

Reply:
Done